CoCSharp/Network/Messages/AllianceStream/ChatStreamEntry.cs

```diff
@@ -0,0 +1,102 @@
+﻿using CoCSharp.Logic;
+using System;
+
+namespace CoCSharp.Network.Messages.AlllianceStream
+{
+    /// <summary>
+    /// Stream entry sent to give the client to comunicate in clan
+    /// entry.
+    /// </summary>
+    public class ChatStreamEntry : AllianceStreamEntry
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ChatStreamEntry"/> class.
+        /// </summary>
+        public ChatStreamEntry()
+        {
+            //space
+            Unknown1 = 3;
```

Comment:
We're not suppose to initialize anything in the default constructor of a Message/StreamEntry.